theSrc/scripts/components/xAxis.js

```diff
@@ -105,7 +105,7 @@ class XAxis extends BaseComponent {
       const columnWidth = this.bounds.width / this.labelCount
       const { sortedWeightedSums } = this.palmMath.getData()
       sortedWeightedSums.forEach(({ name }, i) => {
-        this.container.select(`.xaxis-label[data-name="${name}"]`)
+        this.container.select('.xaxis-label[data-name="' + name.replace(/["\\]/g, '\\$&') + '"]')
```

Comment:
Apparently $& usage is non-standard: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/lastMatch